docs/samples/Microsoft.ML.Samples/Trainers.cs

```diff
@@ -125,11 +127,11 @@ public static void FastTreeRegression()
             var dataWithPredictions = model.Transform(testData);
             var metrics = regressionContext.Evaluate(dataWithPredictions, r => r.label, r => r.score);
 
-            Console.WriteLine($"L1 - {metrics.L1}");
-            Console.WriteLine($"L2 - {metrics.L2}");
-            Console.WriteLine($"LossFunction - {metrics.LossFn}");
-            Console.WriteLine($"RMS - {metrics.Rms}");
-            Console.WriteLine($"RSquared - {metrics.RSquared}");
+            Console.WriteLine($"L1 - {metrics.L1}"); // 3.0035
+            Console.WriteLine($"L2 - {metrics.L2}"); // 12.9096
+            Console.WriteLine($"LossFunction - {metrics.LossFn}"); // 12.9096
+            Console.WriteLine($"RMS - {metrics.Rms}"); // 3.5929
+            Console.WriteLine($"RSquared - {metrics.RSquared}"); // 0.7686
```

Comment:
The exact output metrics will change in time as we change how the algos work, and the default hyperparameters. Hence these values will get stale in time. I would recommend not including them.